src/resources.js

```diff
@@ -1,6 +1,4 @@
 "use strict";
```

Comment:
I consider it normal, yes. See [MDN article on Strict mode](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Strict_mode)